internal/router/controllers/user.go

```diff
@@ -183,148 +179,74 @@ func (c *UserController) GetTableRow(rw http.ResponseWriter, r *http.Request) {
 				})
 			}
 		}
-		out = md
+		rw.WriteHeader(http.StatusOK)
+		_ = json.NewEncoder(rw).Encode(md)
 	default:
-		out = rows
+		opts, err := formatterOptions(r)
+		if err != nil {
+			rw.WriteHeader(http.StatusBadRequest)
+			msg := fmt.Sprintf("Invalid formatting params: %v", err)
+			_ = json.NewEncoder(rw).Encode(errors.ServiceError{Message: msg})
+			log.Ctx(r.Context()).Error().Err(err).Msg(msg)
+			return
+		}
+		formatted, config, err := formatter.Format(res, opts...)
+		if err != nil {
+			rw.WriteHeader(http.StatusInternalServerError)
+			msg := fmt.Sprintf("Error formatting data: %v", err)
+			_ = json.NewEncoder(rw).Encode(errors.ServiceError{Message: msg})
+			log.Ctx(r.Context()).Error().Err(err).Msg(msg)
+			return
+		}
+		if config.Unwrap && len(res.Rows) > 1 {
+			rw.Header().Set("Content-Type", "application/jsonl+json")
+		}
+		rw.WriteHeader(http.StatusOK)
+		_, _ = rw.Write(formatted)
```

Comment:
Ok so this is handling the `GetTableRow` requests when `erc721` format is NOT requested. So we want to use our general formatting support here. `formatterOptions(r)` returns a slice of formatter options derived from the request query parameters. The call to `Format()` returns the formatted data along with the final configuration that was used to format the data. This config is essentially a composite of the default values with the user provided options. It can be used to make subsequent decisions based on the data formatting. On line 201, we make one of these decisions if the data is "unwrapped" and contains more than one row, we set the content type for `jsonl`.